web/packages/teleterm/src/ui/services/workspacesService/documentsService/types.ts

```diff
@@ -105,6 +105,18 @@ export interface DocumentGateway extends DocumentBase {
   origin: DocumentOrigin;
 }
 
+export interface DocumentGatewayKube extends DocumentBase {
+  kind: 'doc.gateway_kube';
+  // rootClusterId and leafClusterId are tech debt. They could be read from targetUri, but
+  // useDocumentTerminal expects these fields to be set on the doc.
+  rootClusterId: string;
+  leafClusterId: string | undefined;
+  gatewayUri?: uri.GatewayUri;
```

Comment:
We don't need `gatewayUri` either. In `DocumentGateway`, it is used to perform operations on the gateway (such as changing the port). AFAIK we won't have to perform such operations on the kube gateway.